docs/tempo/website/operations/deployment.md

```diff
@@ -32,4 +32,45 @@ Helm charts are available in the grafana/helm-charts repo:
 ## Kubernetes manifests
 
 You can find a collection of Kubernetes manifests to deploy Tempo in the [operations/kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests) folder.
-These are generated using the Tanka / Jsonnet.
\ No newline at end of file
+These are generated using the Tanka / Jsonnet.
+
+# Deployment Scenarios
+
+Tempo can be deployed in three modes.
+
+* Single Binary
+* Scalable Single Binary
+* Microservices
+
+These modes are controlled by the runtime configuration `target`, or using the CLI flag `-target`.  The default target is `all`, which we refer to as "Single Binary".
+
+## Single Binary
+
+This mode is the simplest to get started, in which all of the top-level components, referenced in the [Architecture](architecture) section, are run within the same instance of Tempo.
```

Comment:
does this link work? I've been using that "relref" tag. Something like:

`[tenant index]({{< relref "../operations/polling" >}})`